tests/garage/tf/policies/test_categorical_mlp_policy_with_model.py

```diff
@@ -0,0 +1,49 @@
+"""This script creates a unittest that tests qf-derived policy."""
+import pickle
+
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.policies import CategoricalMLPPolicyWithModel
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyDiscreteEnv
+
+
+class TestCategoricalMLPPolicyWithModel(TfGraphTestCase):
+    def setUp(self):
+        super().setUp()
+        self.env = TfEnv(DummyDiscreteEnv())
+        self.policy = CategoricalMLPPolicyWithModel(env_spec=self.env.spec)
+        self.env.reset()
+
+    def test_categorical_mlp_policy_with_model_get_action(self):
```

Comment:
you dont need to include the class names in these methods, e.g. `def test_get_action(self)` is fine